numpy/linalg/lapack_lite/f2c.h

```diff
@@ -214,4 +214,108 @@ typedef doublereal E_f;	/* real function with -R not specified */
 #undef unix
 #undef vax
 #endif
+
```

Comment:
Should add a comment here explaining that these are messy because they are generated from the C file (and explain how to generate them)

Reply:
As explained below, these are copypasted from `libf2c`, and the compiler will detect (and does detect) any missing/incorrect definitions (see all of the new warnings that the compiler previously missed). I this case, I don't believe that time should be spent on the aesthetic issue, but rather the compiler warnings (and in another PR).